FUTURE-NATIVE.md

```diff
@@ -0,0 +1,29 @@
+Lots of people have asked about reason-react compiled to native (Reason Native React Native or RNRN 😎), here's a rough outline of dependencies that have to fall into place.
+
+# Prerequisites
+status: in progress
+
+The main thing here is getting a package manager + build toolchain that make cross-compilation a breeze. @jordwalke and some others are working on this right now, and it's called [esy](https://github.com/esy/esy).
+
+Time remaining before it's usable: 2-8 months? :P [TODO put in a better estimate about timeline]
+
+# Potential direction #1
+status: not started
+
+Use existing react-native infrastructure, and make a reason-react implementation that talks the same protocol as react-native currently uses.
+
+Benefits: we can re-use all the flex-box, native view bindings, etc. that react-native already has.
+Downsides: that might come with some cruft as well.
+
+# Potential direction #2
```

Comment:
Actually I think that the step before is common for the direction number one and two. 

Brisk is separate from any platform specific things. The ambition for the project is to make it useful both when you want to use RN's `MessageQueue`, Terminal, or maybe the Mac. We're striving for the right abstraction level to be totally platform agnostic.